builtin/providers/aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -235,7 +235,7 @@ func TestResourceAWSElastiCacheReplicationGroupIdValidation(t *testing.T) {
 			ErrCount: 1,
 		},
 		{
-			Value:    randomString(17),
+			Value:    randomString(21),
```

Comment:
ErrCount: 1 means that we check that an error is thrown from the validateFunc :)